examples/links/thermal_twinbuilder.py

```diff
@@ -817,6 +817,7 @@ def temperaturesHousingAmbient():
 # Create the e8 input file if it does not exist already
 if Path(inputMotFilePath).exists() == False:
     motorcad = pymotorcad.MotorCAD()
+    motorcad.set_variable("MessageDisplayState", 2)
```

Comment:
Should this be in this PR? I think this is the twinbuilder thermal example.

Reply:
Removing this